packages/bot-web-ui/src/components/quick-strategy/quick-strategy.scss

```diff
@@ -18,6 +18,22 @@
     }
 }
 
+.dc-input {
+    .dc-input__field {
+        &:focus,
+        &:disabled,
+        &:not(:focus):not([value='']) {
+            outline: none;
+
+            & ~ label {
+                transform: translate(0, -1.8rem) scale(0.875);
+            }
+        }
+    }
+    .quick-strategy__label {
```

Comment:
Since we've created a new class now, this shouldn't be inside `.dc-input`.

Reply:
it should be there as we don't want to use` !important`. Parent class gives us a more specific selector

Reply:
Shouldn't be necessary you can create a `quick-strategy__input` classname under while you can add `--field` and `--label`.